settings/mapping.yml

```diff
@@ -105,4 +105,33 @@ tables:
       railway:
         - station
         - platform
-    type: polygon
\ No newline at end of file
+    type: polygon
+   aeroways:
+    columns:
+     - name: osm_id
+       type: id
+     - name: geometry
+       type: geometry
+     - key: name
+       name: name
+       type: string
+     - key: iata
+       name: iata
+       type: string
+     - key: icao
+       name: icao
+       type: string
+     - key: ele
+       name: elevation
+       type: string
```

Comment:
Shouldn't this be a number? https://wiki.openstreetmap.org/wiki/Key:ele

Reply:
Apparently there is only support for integers, but ele would be `NUMERIC`: https://imposm.org/docs/imposm3/latest/mapping.html#column-types